src/fusor/translator.py

```diff
@@ -127,9 +127,12 @@ def _get_gene_element_unnormalized(self, symbol: str) -> GeneElement:
         """
         return GeneElement(
             gene={
-                "id": f"gene:{symbol}",
-                "label": symbol,
-                "type": "Gene",
+                "primaryCoding": {
+                    "id": f"gene:{symbol}",
+                    "code": f"gene:{symbol}",
```

Comment:
Both `code` and `system` are required: https://github.com/ga4gh/gks-core/blob/20e1f153aa546cdaa7a2d2311c91cf54eb10987d/schema/gks-core/json/Coding#L47